tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "target": "ES3",
+    "module": "commonjs",
+    "sourceMap": true
+  }
```

Comment:
It would be good to include a `src` directory option here to target only the typescript directory